friends/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import FriendsList from './components/FriendsList';
+import './App.css';
+
+class App extends Component {
+	constructor() {
+		super();
+		this.state = {
+			data: [],
+			name: '',
+			age: '',
+			email: '',
+		};
+	}
+	componentDidMount() {
+		axios.get('http://localhost:5000/friends')
+      .then( response => this.setState({ data: response.data }))
+      .catch(err => console.log(err))
+  }
+	
+
+  inputChangeHandler = e => {
+    this.setState({ [e.target.name]: e.target.value });
+
+    }
+    submitNewFriend = e => {
+        e.preventDefault();
+    
```

Comment:
This line is [destructuring](https://stackoverflow.com/questions/42307735/destructuring-state-in-react-render) your state, or pulling off the name, age, and email so that down below inside your post request you can simply pass in { name, age, email } in place of what's on lines 33-35.
```suggestion
    const { name, age, email } = this.state;
```